lib/react.dart

```diff
@@ -1584,6 +1588,7 @@ class SyntheticEvent {
   /// See: <https://developer.mozilla.org/en-US/docs/Web/API/Event/type>
   final String type;
 
+  @Deprecated(_syntheticEventDeprecationMessage)
```

Comment:
It'd be helpful to say which APIs specifically to switch to, so that they can just autocomplete/auto-import them. Right now, they'd have to dig through the source code to find them

> Use `createSyntheticEvent` instead.

Reply:
True, good point!